src/hooks/useEdgeSpacing/index.ts

```diff
@@ -0,0 +1,19 @@
+import {useMemo} from 'react';
+import type {EdgeInsets} from 'react-native-safe-area-context';
+import useStyledSafeAreaInsets from '@hooks/useStyledSafeAreaInsets';
+import defaultEdgeSpacing from './defaultEdgeSpacing';
+
+function useEdgeSpacing(padding: Partial<EdgeInsets> = {}): EdgeInsets {
```

Comment:
To avoid recreated the `useMemo`
```suggestion
function useEdgeSpacing(padding: Partial<EdgeInsets> = CONST.EMPTY_OBJECT): EdgeInsets {
```